src/shipit_code_coverage/shipit_code_coverage/chunk_mapping.py

```diff
@@ -14,52 +15,174 @@
 logger = get_logger(__name__)
 
 
+PLATFORMS = ['linux', 'windows']
+# TODO: Calculate this dinamically when https://github.com/klahnakoski/ActiveData-ETL/issues/40 is fixed.
+TEST_COVERAGE_SUITES = ['reftest', 'web-platform', 'mochitest', 'xpcshell', 'jsreftest', 'crashtest']
+
+
+def get_suites(revision):
+    r = requests.post('https://activedata.allizom.org/query', json={
+        'from': 'unittest',
+        'where': {'and': [
+            {'eq': {'repo.branch.name': 'mozilla-central'}},
+            {'eq': {'repo.changeset.id12': revision[:12]}},
+            {'or': [
+                {'prefix': {'run.key': 'test-linux64-ccov'}},
+                {'prefix': {'run.key': 'test-windows10-64-ccov'}}
+            ]}
+        ]},
+        'limit': 500000,
+        'groupby': ['run.suite']
+    })
```

Comment:
We could convert these queries to ESFrontLine in the future

Reply:
Indeed! Performance is not a great concern here, but we could remove some complexity (e.g. we wouldn't need to perform multiple queries to get around the 20k limit)